tests/integration/fixtures/relay.py

```diff
@@ -133,7 +134,7 @@ def inner(
                 default_opts.setdefault(key, {}).update(options[key])
 
         dir = config_dir("relay")
-        dir.join("config.yml").write(json.dumps(default_opts))
+        dir.join("config.yml").write(yaml.dump(default_opts))
```

Comment:
There are few things here:
* first of all I think we should use Yaml for testing the configuration, since we use Yaml config file to configure the service (not JSON) - so we can test this as closer to real world 
* secondly, and which is more relevant to this change - that with new additions to the configuration we have to have the integers as the keys in the map, but JSON can only have strings there. 

And here is the thing, we if want to also support JSON we will have to relay on parsing the strings, maybe in addition to numbers (?),  and getting integers from there.  What do you think?

Reply:
The integer keys make sense. To maintain JSON compatibility, we could also change the format to a list (like below), but then we would have to validate that there are no duplicate entries. So I think sticking with yaml makes sense.

```yaml
ranges:
    - start_at: 0
      name: "ingest-metrics-1"
      config: "metrics_1"
    - start_at: 25000
      name: "ingest-metrics-2"
      config: "metrics_2"
```